### ISSUE TITLE
Can't get it to work

### ISSUE THREAD

**Issue by RienduPre** [hjdhjd/homebridge-doorbird#16]
Hi,

I did everything as discribed in the readme
The motion sensor and doorbell sensor do not give any notification allthough it's enabled.

Please help me troubleshoot this
![img_0053](https://user-images.githubusercontent.com/9009949/30242887-9eb44580-959f-11e7-9ad1-2919c70ea118.PNG)

**Comment by brownad** [comment]
Did you register the endpoint to the doorbird api as mentioned in the docs, it needs the ip of your running homebridge to send updates. Also you can test it by invoking the same endpoint

**Comment by RienduPre** [comment]
Hi,
I made a type in the api commands :-(.
Now I get the notification on my iphone :-)

But in the readme it's stated 
"If placed in a room with a camera it sends a notification with a snapshot to your iOS device(s) after the user activates the DoorBird's doorbell or motion is detected."

I don't see the snapshot, can you sent me a picture how this should like like on the phone ?

Thx,
Rien

**Comment by brownad** [comment]
Did you set the snapshot URL for the ffmpeg plugin correctly? There are examples on that repos wiki for doorbird.  Images are served by the camera due to the plugin picking up ‘motion’ - nothing this doorbird plugin does to handle that.

Drop the config here if you want

**Comment by RienduPre** [comment]
Hi,

another type in that line :-(  I have to quit drinking :-)

It's all working now, super plugin !!!!

One comment though,

The Doorbell accessoire is now off the type motion sensor in HomeKit, but there is also a HomeKit accessoire type DoorBell since IOS10. Any reason why the Doorbell type isn't used in the plugin?

Many thanks in advance,
Rien

**Comment by brownad** [comment]
Hey dude,

Drinking and automation is the perfect combination 😎

Glad it’s working for you.

Yes I did want to use the doorbell type but it’s more work than I imagined it requires doing extra work on top of the ffmpeg camera plugin to add a doorbell service.

I haven’t spent time to try it fully maybe someone else can crack it and merge it in here.  Also the doorbell kept changing so I gave up.

All the best.
